### DISCUSSION TITLE
iTwin.js 3.x End of Life approaching

### DISCUSSION THREAD

**Discussion by JiriJa** [iTwin/itwinjs-core#6435] [Q&A]
Hello Everybody,

Please can I ask You? As mentioned via announcement „**Please upgrade your client to 4.x**”, please is anywhere „**step by step**“ manual how to do it? 

Best Regards.
Jiri

**Comment by aruniverse** [comment]
Please review the changelogs for the following
- [iTwin.js Core](https://github.com/iTwin/itwinjs-core/releases/tag/release/4.0.0)
- [AppUI](https://github.com/iTwin/appui/blob/master/docs/changehistory/4.0.0.md)
- [Presentation](https://github.com/iTwin/presentation/blob/master/packages/components/CHANGELOG.md#400)
- [Viewer](https://github.com/iTwin/viewer/blob/master/releases/CHANGELOG-4.0.md)

**Reply by JiriJa** [reply]
Hello Arun, thanks for Your reply. I´m going to investigate your links.
